openmc/mgxs/mgxs.py

```diff
@@ -1351,8 +1360,6 @@ def build_hdf5_store(self, filename='mgxs.h5', directory='mgxs',
                 std_dev = self.get_xs(subdomains=[subdomain], nuclides=[nuclide],
                                       xs_type=xs_type, value='std_dev',
                                       row_column=row_column)
-                average = average.squeeze()
```

Comment:
The `get_xs(...)` method now has a squeeze parameter that defaults to true, so this would be redundant.